src/main/java/seedu/address/logic/commands/StatisticsCommand.java

```diff
@@ -18,38 +18,78 @@ public class StatisticsCommand extends Command {
 
     public static final String COMMAND_WORD = "statistics";
 
-
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        // create a filtered list of all the predicate persons
-        // then create iteration
         ObservableList<Person> lastShownList = model.getFilteredPersonList();
         FilteredList<Person> fullContactList = new FilteredList<>(lastShownList);
-        int numOfApplicants = 0;
-        HashMap<JobCode, Integer> jobDict = new HashMap<JobCode, Integer>();
-        HashMap<Tag, Integer> tagDict = new HashMap<Tag, Integer>();
+
```

Comment:
Good use of FilteredList to handle the display of persons. However, consider checking if lastShownList might be empty and handling that case appropriately to avoid potential issues later in the code.